release-notes.md

```diff
@@ -35,6 +35,9 @@ These release notes track our current efforts to document changes to the F# proj
 * `SynMeasure` was extended with [SynMeasure.Paren](https://fsharp.github.io/fsharp-compiler-docs/reference/fsharp-compiler-syntax-synmeasure.html#Paren) case.
 * Dynamic expressions (like `x?y`) are now represented as [SynExpr.Dynamic](https://fsharp.github.io/fsharp-compiler-docs/reference/fsharp-compiler-syntax-synexpr.html#Dynamic) in the Untyped Syntax Tree.
 * Members with `get` and/or `set` are now represented as [SynMemberDefn.GetSetMember](https://fsharp.github.io/fsharp-compiler-docs/reference/fsharp-compiler-syntax-synmemberdefn.html#GetSetMember) in the Untyped Syntax Tree.
+* `DoesIdentifierNeedBackticks` is removed, it should always be sufficient to call `NormalizeIdentifierBackticks` or else call something in `PrettyNaming`
```

Comment:
@auduchinok @baronfel  Can you check this?  We think it is OK but would like to know if Rider or VSCode are depending on these

Reply:
So there will be followups on that PR, if we find issues we can revert some of these changes in those followups

Reply:
@dsyme It seems it won't affect us. I'll raise an issue if anything doesn't work as expected.